lib/kamal/commands/lock.rb

```diff
@@ -41,7 +41,9 @@ def stat_lock_dir
     end
 
     def lock_dir
-      "#{config.run_directory}/lock-#{config.service}"
+      dir_name = [ config.service, config.destination ].compact.join("-")
+
+      "#{config.run_directory}/lock-#{dir_name}"
```

Comment:
Could we put the locks in `"#{config.run_directory}/locks/#{dir_name}"`? And also use `File.join` to create the path?

We'll need `mkdir -p` when making the directory if we do that.

Reply:
Yes, having `locks` folder seems to be a better option.